### PR TITLE
feat(FN-1290): store reporting schedule against banks

### PR DESCRIPTION
**Pull request by francescastocco**
## Introduction :pencil2:
We need to know the reporting schedule for each bank

## Resolution :heavy_check_mark:
- Added reporting schedule to mock banks
- Added reporting schedule to db model

### CODE REVIEW

File: dtfs-central-api/src/types/db-models/banks.d.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import { WithId } from 'mongodb';
 
+type ReportPeriodSchedule = {
```
Comment (JoshBinns2000) [inline dtfs-central-api/src/types/db-models/banks.d.ts:3]: I think this should be `UtilisationReportPeriodSchedule`

**Review by JoshBinns2000 — COMMENTED** [review]
Looks good code-wise. Only one comment re type name

File: utils/mock-data-loader/banks/index.js [hunk 2]
```diff
@@ -11,6 +11,20 @@ const BANKS = [
...
+      { startMonth: 4, endMonth: 4 },
+      { startMonth: 5, endMonth: 5 },
+      { startMonth: 6, endMonth: 6 },
+      { startMonth: 7, endMonth: 7 },
+      { startMonth: 8, endMonth: 8 },
+      { startMonth: 9, endMonth: 9 },
+      { startMonth: 10, endMonth: 10 },
+      { startMonth: 11, endMonth: 11 },
+      { startMonth: 12, endMonth: 12 },
+    ],
```
Comment (toddym42) [inline utils/mock-data-loader/banks/index.js:27]: Could save this as a `MONTHLY_REPORT_PERIOD_SCHEDULE` or similar to simplify adding this everywhere.

**Review by toddym42 — APPROVED** [review]

File: dtfs-central-api/src/types/db-models/banks.d.ts (hunk 1 above)
Comment (francescastocco) [reply dtfs-central-api/src/types/db-models/banks.d.ts:3]: I thought about it but if we stored any other reporting schedules they could also be the same type

**Review by toddym42 — APPROVED** [review]

**Pull request merged**